container/stream/attach.go

```diff
@@ -33,33 +33,47 @@ type AttachConfig struct {
 	// For example, this would close the attached container's stdin.
 	CloseStdin bool
 
+	UseStdin, UseStdout, UseStderr bool
+
+	CStdin           io.WriteCloser
```

Comment:
Please add a comment about `{Use,C}Std{in,out,err}` here for ease of understandability?